sh/tests/check-user_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted="./check-user.sh $@
+    "
+    expected="./check-user.sh $@
+    "
+    submitted+=$(2>&1 bash "$script_dirS"/student/check-user.sh "$@")
```

Comment:
not sure why but as soon as i format this changes to 
```
submitted+=$(bash 2>&1 "$script_dirS"/student/check-user.sh "$@")
expected+=$(bash 2>&1 "$script_dirS"/solutions/check-user.sh "$@")
```
But it works the same way so no worries here

Reply:
I've applied the autoformat, but yes it seems like a convention, not changing much